NEMO/interlocks.py

```diff
@@ -201,6 +201,44 @@ def _send_command(self, interlock: Interlock_model, command_type: Interlock_mode
 			sock.close()
 
 
+class ProXrInterlock(Interlock):
+	"""Very basic support for ProXR relay controllers.
+	Supports on/off for all relays in bank 1.
+	See https://ncd.io/proxr-quick-start-guide/ for more about ProXR.
+	"""
+	# supported proxr commands
+	PXR_CMD_B1_OFF = (254, 129, 1)
+	PXR_CMD_B1_ON = (254, 130, 1)
+
+	def _send_bytes(self, relay, proxrcmd):
+		"""Returns the response from the relay controller.
+		Argument relay is a connected socket object.
+		Argument proxrcmd (the ProXR command) is an iterable of 8-bit integers.
+		"""
+		# make sure that all bytes get sent
+		bytes_sent = 0
+		while bytes_sent < len(proxrcmd):
+			bytes_sent += relay.send(bytes(proxrcmd[bytes_sent:]))
+		# relay responses can include erroneous data
+		# only the last byte of the response is important
+		return relay.recv(64)[-1]
```

Comment:
does the response contain any information on whether the command was successful? does it contain any extra information if it's not successful? that would be interesting to have.